internal/terraform/exec/exec_mock_test.go

```diff
@@ -13,3 +17,82 @@ func TestMain(m *testing.M) {
 
 	os.Exit(m.Run())
 }
+
+func TestExec_timeout(t *testing.T) {
```

Comment:
These tests were moved out of `exec_test.go` because I felt it was just testing the mock.

Reply:
yeah i think you can just drop them